src/stuf/stuf_zds/models.py

```diff
@@ -21,14 +21,34 @@ class StufZDSConfig(SingletonModel):
         related_name="stuf_zds_config",
         null=True,
     )
+    zaakbetrokkene_registrator_omschrijving = models.CharField(
+        _("description for registrator zaakbetrokkene"),
+        max_length=100,
+        default="",
+        blank=True,
+        help_text=_(
+            "The 'zaakbetrokkene.omschrijving' value for the employee who registers "
+            "the case on behalf of the customer."
+        ),
+    )
     zaakbetrokkene_partners_omschrijving = models.CharField(
         _("description for zaakbetrokkene partners registration"),
         max_length=100,
         default="",
         blank=True,
         help_text=_(
-            "The description that will be added if the zaakbetrokkene registration is "
-            "used for the partners."
+            "The 'zaakbetrokkene.omschrijving' value for when the partners of the "
+            "form submitter are registered as 'betrokkene'."
```

Comment:
Is `betrokkene` correct?Shouldn't it be `zaakbetrokkene`?

Reply:
in this context they're synonyms :) zaakbetrokkene is just a bit more specific